common/app/model/DatesAndTimes.scala

```diff
@@ -119,7 +119,9 @@ object GUDateTimeFormatNew {
       case _ => date.toString(DateTimeFormat.forPattern("HH.mm z").withZone(timezone))
     }
   }
-  def dateTimeToLiveBlogDisplay(dateTime: DateTime, timezone: DateTimeZone): String = {
-    dateTime.toString(DateTimeFormat.forPattern("HH:mm z").withZone(timezone))
+  def formatTimeForDisplayNoTimezone(dateTime: DateTime, request: RequestHeader): String = {
+    val edition = Edition(request)
+    val timezone = edition.timezone
+    dateTime.toString(DateTimeFormat.forPattern("HH.mm").withZone(timezone))
```

Comment:
Why is the forPattern arg different here than the one above?

Reply:
The `z` is for the timezone, which we don't wanna include in the string, and I'm waiting on confirmation from @HarryFischer but the style guide suggests we should be using `12.12` rather than 12:12` :)

The previous function `dateTimeToLiveBlogDisplay` wasn't actually being used anywhere, so what I've done is the equivalent of deleting it & adding a new one that meets our new requirements!